jbmc/regression/jbmc/must-not-throw-annotation/org/cprover/MustNotThrow.java

```diff
@@ -0,0 +1,3 @@
+package org.cprover;
+
+public @interface MustNotThrow { }
```

Comment:
Shouldn't this go in an actual org.cprover package? Does such a thing exist?

Reply:
It will, but this is just a test that if such a thing exists then it functions as intended. It's a "mini models library" if you like.

Reply:
The 'mini models library' is here: https://github.com/diffblue/java-models-library/tree/master/src/main/java/org/cprover
Please add it there. It will then be compiled into the org.cprover.jar

Reply:
And additionally, it should also be duplicated in models-library (until models-library moves to extending java-models-library).

Reply:
Huh? That's the real models lib, isn't it? My intent was to submit this to get the functionality in (with a minimal test that doesn't use the models lib, but just mocks up the relevant interface), then ask @allredj for the actual lib support.

Reply:
Oh so the model of the JDK and the `org.cprover` is in the same repo? That's a surprising choice? I can see someone using JBMC and wanting access to `org.cprover.nondetInt()` without wanting to pull down custom versions of `java.lang.Object`.

Reply:
The org.cprover.jar provides the interface to JBMC. It is required to *compile* code that uses your annotation (and assume, nondetX, etc). org.cprover.jar is not required to be on the classpath when running JBMC.

Reply:
I'm also not happy that it is in the java-models-library repo, but since it is required for compiling the models, we would end up with circular submodules.

Reply:
Where's the cycle? Repo one: `org.cprover`, repo two: `core-models` depends on `org.cprover`. No need for `org.cprover` to know about the core models?

Reply:
If you put it into its own repo then there is no cycle.

Reply:
Let's create a separate repo then. As @thk123 suggested, it might even make sense to put it on Maven Central repo, which avoids having a submodule in models-library and also makes it easier to use for JBMC user who want to hack their own models and harnesses.

Reply:
For the avoidance of doubt - this is out of scope for this PR right @peterschrammel? This is fine for now, and can be tidied up when the new repo is created?

Reply:
Yes.